dev/release/03-binary-submit.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -ex
+
+main() {
+    local -r source_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+    local -r source_top_dir="$( cd "${source_dir}/../../" && pwd )"
+    pushd "${source_top_dir}"
+
+    if [ "$#" -ne 2 ]; then
+        echo "Usage: $0 <version> <rc-num>"
+        exit 1
+    fi
+
+    local -r version="$1"
+    local -r rc_number="$2"
+    local -r tag="adbc-${version}"
+    local -r rc_branch="release-${version}-rc${rc_number}"
+
+    : ${REPOSITORY:="apache/arrow"}
+
+    echo "Starting GitHub Actions workflow on ${REPOSITORY}:${rc_branch}"
+
+    gh workflow run \
+       --repo "${REPOSITORY}" \
+       --ref "${rc_branch}" \
+       packaging-wheels.yml \
+       --raw-field upload_artifacts=false
```

Comment:
How about adding the following to `.github/workflows/packaging-wheels.yml` instead of creating and running this file manually?

```diff
diff --git a/.github/workflows/packaging-wheels.yml b/.github/workflows/packaging-wheels.yml
index 19f88e5..6774260 100644
--- a/.github/workflows/packaging-wheels.yml
+++ b/.github/workflows/packaging-wheels.yml
@@ -18,6 +18,11 @@
 name: Packaging - Nightly Artifacts
 
 on:
+  push:
+    branches-ignore:
+      - '**'
+    tags:
+      - '*'
   schedule:
     - cron: "0 0 * * *"
   workflow_dispatch:
```

Reply:
Will do. I'll put that in a separate PR so I can test with it a little more easily.